src/compiler/crystal/syntax/lexer.cr

```diff
@@ -1990,8 +1990,8 @@ module Crystal
             char = next_char
             next_char
             @token.type = :STRING
-            if string_end == '/' && char == '/'
-              @token.value = "/"
+            if string_end == '/' && (char == '/' || char.ascii_whitespace?)
```

Comment:
Why should it depend on string_end? Will `%r(\ )` differ?? Needs spec for that too anyway

Reply:
Because this this fix is needed to other delimiters. And, I respects existing process for `\/`.

Reply:
I think we should make it the same regardless of delimiter. Otherwise it might be a bit confusing.

Reply:
@asterite You added this at 19fdf45... Did you change your mind?

Reply:
I don't know what happened 🤷

Reply:
Ah, it is problematic. For example, this output should be `true`, but `true` is shown:

```cryystal
macro identity(x)
  {{x}}
end

p %r(\/) == identity(%r(\/))
```

I'll fix. Ok?